File: Fibonacci.cpp

#include <iostream>
using namespace std;

int main()
{
    int n, t1 = 0, t2 = 1, next_term = 0;
    cout << "\nEnter the number of terms: ";
    cin >> n;
    cout << "\n";
    for (int i = 1; i <= n; ++i)
    {
        if (i == 1)
        {
            cout << i << "\t" << t1 << "\n";
            continue;
        }
        if (i == 2)
        {
            cout << i << "\t" << t2 << "\n";
            continue;
        }
        next_term = t1 + t2;
        t1 = t2;
        t2 = next_term;
        cout << i << "\t" << next_term << "\n";
    }
    cout << "\n[C++]\n\n";
    return 0;
}

File: Fibonacci.py
n = int(input("\nEnter the number of terms: "))

fibonacci = [0, 1]

for i in range(2, n):
    fibonacci.append(fibonacci[i - 1] + fibonacci[i - 2])

print()

for i, value in enumerate(fibonacci):
    print(i + 1, "\t", value)

print("\n[Python]\n")
